content/en/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Check whether Dockershim deprecation affects you
+content_type: task 
+reviewers:
+- SergeyKanzhelev
+weight: 20
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.20" state="deprecated" >}}
+
+The `dockershim` component of Kubernetes allows to use Docker as a Kubernetes's
+{{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}.
+Kubernetes' built-in `dockershim` component was deprecated in release v1.20.
+
+This page explains how your cluster could be using Docker as a
+{{< glossary_tooltip text="container runtime" term_id="container-runtime" >}},
```

Comment:
nit: The container runtime glossary tooltip is used twice.

You could add links to the sections that follow (and create custom anchor text to link to). Something like:
```
This page [explains the role of `dockershim`](#role-of-dockershim) and provides
 [instructions to check for `dockershim` dependencies](#find-docker-dependencies).
```